services/src/main/java/org/keycloak/services/resources/admin/UserResource.java

```diff
@@ -268,6 +268,7 @@ public static void updateUserFromRep(UserProfile profile, UserModel user, UserRe
 
         if (rep.isEnabled() != null) user.setEnabled(rep.isEnabled());
         if (rep.isEmailVerified() != null) user.setEmailVerified(rep.isEmailVerified());
+        if (rep.getCreatedTimestamp() != null) user.setCreatedTimestamp(rep.getCreatedTimestamp());
```

Comment:
The problem here is that the `updateUserFromRep` is called also during updates. We don't want to set this attribute but only during creation (or importing).

Reply:
I don't see why we shouldn't let a REST api add a value to this. We are checking for non null.

Reply:
It surely does. But during creation not during updates. Making sure this attribute is only set during creation solves your problem, right?

Reply:
I guess. But I'm also not sure why a later REST call in an update to (fix?) the create timestamp should be dis-allowed.

Reply:
🤔 how do I go about discerning if this is a create vs an update?

Reply:
I did add a change to only do on creations and added a test case.